applications/StructuralMechanicsApplication/custom_elements/truss_element_3D2N.hpp

```diff
@@ -1,191 +1,196 @@
-
-
-		~TrussElement3D2N() override;
-
-
-		BaseType::Pointer Create(
-			IndexType NewId,
-			NodesArrayType const& rThisNodes,
-			PropertiesType::Pointer pProperties) const override;
-
-		void EquationIdVector(
-			EquationIdVectorType& rResult,
-			ProcessInfo& rCurrentProcessInfo) override;
-
-		void GetDofList(
-			DofsVectorType& rElementalDofList,
-			ProcessInfo& rCurrentProcessInfo) override;
-
-		void Initialize() override;
-
-		MatrixType CreateElementStiffnessMatrix();
-
-		void CalculateOnIntegrationPoints(
-			const Variable<double>& rVariable,
-			std::vector<double>& rOutput,
-			const ProcessInfo& rCurrentProcessInfo) override;
-
-		void GetValueOnIntegrationPoints(
-			const Variable<double>& rVariable,
-			std::vector<double>& rValues,
-			const ProcessInfo& rCurrentProcessInfo) override;
-
-		void UpdateInternalForces(
-			VectorType& rinternalForces);
-		void CreateTransformationMatrix(
-			Matrix& rRotationMatrix);
-		double CalculateCurrentLength();
-
-		void CalculateOnIntegrationPoints(
-			const Variable<Vector>& rVariable,
-			std::vector<Vector>& rOutput,
-			const ProcessInfo& rCurrentProcessInfo) override;
-
-		void GetValueOnIntegrationPoints(
-			const Variable<Vector>& rVariable,
-			std::vector<Vector>& rValues,
-			const ProcessInfo& rCurrentProcessInfo) override;
-
-		void CalculateLocalSystem(
-			MatrixType& rLeftHandSideMatrix,
-			VectorType& rRightHandSideVector,
-			ProcessInfo& rCurrentProcessInfo) override;
-
-
-		void CalculateRightHandSide(
-			VectorType& rRightHandSideVector,
-			ProcessInfo& rCurrentProcessInfo) override;
-
-		void CalculateLeftHandSide(
-			MatrixType& rLeftHandSideMatrix,
-			ProcessInfo& rCurrentProcessInfo) override;
-
-		void CalculateMassMatrix(
-			MatrixType& rMassMatrix,
-			ProcessInfo& rCurrentProcessInfo) override;
-
-		void CalculateDampingMatrix(
-			MatrixType& rDampingMatrix,
-			ProcessInfo& rCurrentProcessInfo) override;
-
-
-		void AddExplicitContribution(const VectorType& rRHSVector,
-			const Variable<VectorType>& rRHSVariable,
-			Variable<array_1d<double, 3> >& rDestinationVariable,
-			const ProcessInfo& rCurrentProcessInfo) override;
-
-
-		void GetValuesVector(
-			Vector& rValues,
-			int Step = 0) override;
-
-		void GetSecondDerivativesVector(
-			Vector& rValues,
-			int Step = 0) override;
-
-		void GetFirstDerivativesVector(
-			Vector& rValues,
-			int Step = 0) override;
-
-		int  Check(
-			const ProcessInfo& rCurrentProcessInfo) override;
-
-
-		double CalculateGreenLagrangeStrain();
-		double CalculateReferenceLength();
-
-		VectorType CalculateBodyForces();  
-
-		bool ReturnIfIsCable();
-
-
-
-	private:
-		bool mIsCompressed;
-		bool mIsLinearElement = false;
-
-		TrussElement3D2N() {};
-
-		friend class Serializer;
-		void save(Serializer& rSerializer) const override;
-		void load(Serializer& rSerializer) override;
-	};
-
-
-}
-
-
-#endif
+// ==============================================================================

```

Comment:
Would you please use the new [header template](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/templates/header_template)?